GLAA.Domain/IDeletable.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace GLAA.Domain
+{
+    public interface IDeletable
+    {
+        bool Deleted { get; set; }
+        DateTime? DateDeleted { get; set; }
```

Comment:
Do we need to capture the user who deleted the entity as well?

Reply:
I would have thought so. I didn't want to put too much extra in at this stage though until the audit trail side of things has been spec'ed.

Reply:
Good thinking, happy to leave it as is for the time being